mobile/js_files/package.json

```diff
@@ -38,7 +38,7 @@
     "react-native-screens": "^1.0.0-alpha.23",
     "react-native-shake": "^3.3.1",
     "react-native-splash-screen": "^3.2.0",
-    "react-native-status-keycard": "^2.5.14",
+    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#feature/exportKeyWithPath",
```

Comment:
use a tag before you merge

Reply:
please create a tag so we can merge this PR

Reply:
I've created and pushed a `v2.5.16` tag into `react-native-status-keycard`. I don't understand what should i put into `package.json` now. Is there some additional step required in order to be able to write "react-native-status-keycard": "^2.5.16" here?

Reply:
you would have to push it to npm I suppose, but in the meantime you can just refer to the tag instead of the branch

Reply:
Rebased and squashed commits, updated `react-native-status-keycard` version to `^2.5.16`.